keps/sig-cluster-lifecycle/kubeadm/20190424-kubeadm-for-windows.md

```diff
@@ -115,23 +115,35 @@ The motivation for this KEP is to provide a tool to allow users to take a Window
 
 A user will have a Windows machine that they want to join to an existing Kubernetes cluster.
```

Comment:
The user value or user story could be re-framed to be 
"A user can join a Windows VM to an existing Kubernetes Cluster"

Reply:
Somehow Github didn't allow me to select the line that I actually wanted to review, so here it is! 

`A user will have a Windows machine that they want to join to an existing Kubernetes cluster.`

Reply:
> The user value or user story could be re-framed to be
"A user can join a Windows VM to an existing Kubernetes Cluster"

Windows Nodes in a k8s cluster are not necessarily VMs only, thus "machines" is a better generalization.